src/components/NavigationBar/index.js

```diff
@@ -17,19 +19,47 @@ const NavigationBar = () => {
     }
     return (
         <>
-            <Link className={"btn btn-primary"} to={"/home"}>Home</Link>
-            We can have navigation bars here
+        <Navbar bg="light" expand="lg" fixed="top">
+        <Container>
+          <Navbar.Brand href="#landingpage">LPQK Movies</Navbar.Brand>
```

Comment:
What does the ```href``` do here?

Reply:
The href is to navigate to a page. I think bootstrap Navbar doesn't have other route method like Link.

Reply:
But currently, it does not do anything when you click it since we don't have an element whose id is "landingpage", right? I suggest either removing it or adding the id.

Reply:
I was going to ask that during our discussing as to where should the logo take the user too. Does the home page sound good or I can remove the link on it.

Reply:
Removed the href since it was not needed.